climada/hazard/centroids/centr.py

```diff
@@ -538,6 +538,35 @@ def append(self, centr):
                 setattr(self, var_name, np.append(var_val, centr_val).
                         astype(var_val.dtype, copy=False))
 
+    @staticmethod
+    def join_centroid(centroids_list):
```

Comment:
I suggest to apply the same logic as the built-in `set.union` method (https://docs.python.org/3/library/stdtypes.html#frozenset.union): Don't make it static, but always return a new object. That way you can call it just like a static method if needed. Moreover, why not call it `union`?
```suggestion
    def union(self, *others):
```

Reply:
Yeah I think union is a more precise name, unless there are analogous `join_` methods elsewhere in CLIMADA?

Reply:
I agree with `union` and not making it static is fine for me. Although I personally do not like the logic to have a non-static method that always returns a new object of the same type. This is for me very counter-intuitive.

Reply:
Also, should the method now make the union only of the `others` or include the current `Centroids` object too?

Reply:
Well, if that's counter-intuitive for you, you didn't work with pandas too much, did you? :smile: Mathematical operations are translated in python to be method calls of objects. So, `a + b` is translated to `a.__add__(b)`.  Following that scheme, the operation `A ∪ B` is translated to `A.union(B)`. And naturally, the results are new objects. So, in NumPy, when you have two arrays `x` and `y`, and you write `z = x + y` then this is the same as  `z = x.__add__(y)` and still, `z` is not the same object as `x`. Are you fine with that?

It should work just like the built-in `set.union` in my opinion. So, `self` is of course included.